acceptance/chaos_test.go

```diff
@@ -73,11 +73,10 @@ func TestChaos(t *testing.T) {
 		go func(i int) {
 			r, _ := randutil.NewPseudoRand()
 			value := randutil.RandBytes(r, 8192)
+			k := atomic.LoadInt64(&count)
```

Comment:
as discussed in person, the values written here don't matter. This is mostly an artifact of this having spawned from `TestPut`.